content/en/docs/components/pipelines/author-a-pipeline/components.md

```diff
@@ -165,52 +165,33 @@ To create a containerized component, you must:
 
 ### 3. Custom container components
 
-Container-based components make it easier to use the pre-existed container to build a KFP component. 
-The user will control the container entrypoint and is able to define the component by specifying their container, commands, and args in a `ContainerSpec` object with similar authoring experiences as defining Python components. 
+Custom container components make it easier to use the pre-existed container to build a KFP component. 
```

Comment:
Can you describe what you mean by "the pre-existed container"? I think a slightly more accurate description would be "Custom container components allow you to specify a container to execute as your component." or something similar. WDYT?

Reply:
Yes, I agree that's better phrasing! I will correct that.